WordPress/wordpress-develop
# Twenty Twenty-Two: add alternate style variations

Trac ticket: https://core.trac.wordpress.org/ticket/55433

This PR adds three style variations to the Twenty Twenty-Two theme.  It is dependent on the Webfonts API: https://github.com/WordPress/wordpress-develop/pull/2622. If and when that is merged, this PR can be reviewed.

**Testing instructions**

- Check out this PR
- Go to the site editor and visit the global styles panel
- Click "Browse Styles"
- Verify all four variations load correctly

---

Todo: 

- [x] Get fonts working (Reliant on https://github.com/WordPress/gutenberg/pull/40262)
- [x] Implement pink and red variants.
- [x] Update readme with font information.
